### PR TITLE
Update forsetisecurity.org-dev faqs for VM instances security updates…

### PR DESCRIPTION
**Pull request by kssrini**
Updating forsetisecurity.org-dev faq's for VM instances up to date with security patches.

### CODE REVIEW

File: _faq/installation_and_deployment/forseti VM instance security updates.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: How do I keep Forseti VM instances (client and server) up to date with security patches?
```
Comment (blueandgold) [inline _faq/installation_and_deployment/forseti VM instance security updates.md]: spelling: `up-to-date` is the proper spelling

File: _faq/installation_and_deployment/forseti VM instance security updates.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+title: How do I keep Forseti VM instances (client and server) up to date with security patches? 
+order: 7
+---
+{::options auto_ids="false" /}
+
+
+[GCE](https://cloud.google.com/compute/docs/images) VM instances have the [unattended-upgrades](https://wiki.debian.org/UnattendedUpgrades) tool to automatically update the operating system, software, or security patches from the [Debian security](https://www.debian.org/security/) repository.
+
+However, kernel patches do not take effect until your VM instance is restarted. By default, GCE does not automatically restart running instances.  So you must either restart your instances manually to update the kernel, or [apply the mechanism provided by the unattended-upgrades tool to automatically do the restart](https://wiki.debian.org/UnattendedUpgrades#Automatic_call_via_.2Fetc.2Fapt.2Fapt.conf.d.2F20auto-upgrades).
```
Comment (blueandgold) [inline _faq/installation_and_deployment/forseti VM instance security updates.md]: This line is too long.  Can we break before 80th column?

Also, one space after period.

File: _faq/installation_and_deployment/forseti VM instance security updates.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+---
+title: How do I keep Forseti VM instances (client and server) up to date with security patches? 
+order: 7
+---
+{::options auto_ids="false" /}
+
+
+[GCE](https://cloud.google.com/compute/docs/images) VM instances have the [unattended-upgrades](https://wiki.debian.org/UnattendedUpgrades) tool to automatically update the operating system, software, or security patches from the [Debian security](https://www.debian.org/security/) repository.
```
Comment (blueandgold) [inline _faq/installation_and_deployment/forseti VM instance security updates.md]: Break before 80th column.

File: _faq/installation_and_deployment/forseti VM instance security updates.md (hunk 1 above)
Comment (kssrini) [reply _faq/installation_and_deployment/forseti VM instance security updates.md]: updated

File: _faq/installation_and_deployment/forseti VM instance security updates.md (hunk 3 above)
Comment (kssrini) [reply _faq/installation_and_deployment/forseti VM instance security updates.md]: Done

File: _faq/installation_and_deployment/forseti VM instance security updates.md (hunk 2 above)
Comment (kssrini) [reply _faq/installation_and_deployment/forseti VM instance security updates.md]: Done

**Pull request merged**